<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FriendMapConfigPage</class>
 <widget class="QWidget" name="FriendMapConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>451</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Geo IP Data Path</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Marble Path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="geoip_path_line"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="projectionLabel">
      <property name="text">
       <string>Projection</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="projection_box">
      <item>
       <property name="text">
        <string>Spherical</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Equirectangular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mercator</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="marble_path_line"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
